app/templates/components/modals/add-system-role-modal.hbs

```diff
@@ -0,0 +1,67 @@
+<div class="header">
+  {{t 'Add New System Role'}}
+</div>
+<div class="content">
+  <form class="ui form" id="add-system-role-form" autocomplete="off" {{action 'addRole' on='submit' preventDefault=true}}>
+    <div class="field">
+      <label class="required">
+        {{t 'Name'}}
+      </label>
+      {{input type='text' name='role_name' placeholder=(t 'Enter Role name')}}
+    </div>
+    <div class="field">
+      <label class="required">
+        {{t 'Panels'}}
+      </label>
+      {{#ui-dropdown class="fluid multiple selection" selected=userRoles onChange=(action (mut selectedRoles))}}
```

Comment:
Why is the selected variable different from the variable that is being mutated ?